### ISSUE TITLE
ERR_BLOCKED_BY_CLIENT Chrome error when I configure to open ReadLater in a new tab

### ISSUE THREAD

**Issue by chelder86** [napsternxg/ReadLater#38]
* [ ] ReadLater version you are using: 1.6.19
* [ ] Messages from the console of your popup page: I don't see any "inspect popup option" to click. I clicked on "develper mode". Then: "collect errors".
* [ ] Messages from the console of your backgroud page: I only see this: "bg.js:396"

I'd like to show ReadLater on a new tab. I've tried a few extensions for that. Unfortunately, Chrome returns: 

> ffkmmgcnbjamaacfolbfodfpcepapmnp is blockedRequests to the server have been blocked by an extension.
> Try disabling your extensions.
> ERR_BLOCKED_BY_CLIENT

**It also happens on a incognito window.**

Two examples of extensions tried:

- ProductivityTab — Custom New Tab Dashboard: https://chrome.google.com/webstore/detail/productivitytab-%E2%80%94-custom/iccjgbbjckehppnpajnmplcccjcgbdep
- Custom New Tab URL: https://chrome.google.com/webstore/detail/custom-new-tab-url/mmjbdbjnoablegbkcklggeknkfcjkjia

Screenshots:

![image](https://user-images.githubusercontent.com/3020189/98468240-e3e53000-21d9-11eb-8119-6c03c43e8389.png)

![image](https://user-images.githubusercontent.com/3020189/98468280-18f18280-21da-11eb-9b58-e023d23e7dd7.png)

![image](https://user-images.githubusercontent.com/3020189/98468297-2b6bbc00-21da-11eb-9e1a-1758df6ff638.png)

**Comment by gypsicoder** [comment]
@chelder86 
Have you found the solution of the issue?

**Comment by chelder86** [comment]
> Have you found the solution of the issue?

No, I didn't. Maybe that's why I'm not using the extension anymore...

**Comment by gypsicoder** [comment]
It's sad about this.
